packages/web-app-media-viewer/src/App.vue

```diff
@@ -208,7 +208,7 @@ export default {
     },
 
     isActiveMediaFileTypeVideo() {
-      return this.videoExtensions.includes(this.activeMediaFile.extension)
+      return this.videoExtensions.includes(this.activeMediaFile.extension.toLowerCase())
```

Comment:
Might be a low hanging fruit: the `activeMediaFile` should have a field `mimeType` by now. Would be much easier and less error prone to check if the mimetype references a video format. Then you could ditch this one `toLowerCase` case. 😁

Reply:
let's get this PR merged and look into that in another PR ok?

Reply:
Yep, was my gut feeling too!

Reply:
It's ready from my side, go ahead and merge if you think so too!